pulsar-broker/src/main/java/org/apache/pulsar/common/naming/NamespaceBundles.java

```diff
@@ -66,7 +66,7 @@ public NamespaceBundles(NamespaceName nsname, NamespaceBundleFactory factory,
         this.factory = Objects.requireNonNull(factory);
         this.localPolicies = localPolicies;
         checkArgument(partitions.length > 0, "Can't create bundles w/o partition boundaries");
-
+        this.topicBundleAssignmentStrategy = TopicBundleAssignmentFactory.create(factory.getPulsar());
```

Comment:
hi, can this stragegy field move to BundleFactory ?  In heavy traffic cluster, it seems a lot NamespaceBundles wil be created. When init this object `TopicBundleAssignmentFactory.create` will use java reflection multi times which will is rather slow. if this object is just stateless, maybe put in the BundleFactory is more suitable

Reply:
Good catch